packages/p2p/src/components/modal-manager/modals/email-verification-modal/email-verification-modal.jsx

```diff
@@ -81,13 +83,13 @@ const EmailVerificationModal = ({
                     </React.Fragment>
                 )}
             </Modal.Body>
-            {should_show_resend_email_button && should_show_reasons_if_no_email && (
```

Comment:
are there any cases in the future we need to have this check though?

Reply:
this check always returns true and is not being used anywhere that's why i removed it